cli-plugins/apps/how-to-guides/how-to-guide-1.hbs.md

```diff
@@ -0,0 +1,28 @@
+# How-to-Guides simple reference
+
+## <a id='create-from-private-repo'></a> Create a workload from a private Git repository
+
+Check how to setup a workload from a Private Git repository in 
+[Out of the box supply chain with private Git repos](../../../scc/building-from-source.hbs.md#private-gitrepository),
```

Comment:
Are you sure these links are correct? I can't get any of them to work

Reply:
they work for me